tests/test_script_arguments.py

```diff
@@ -71,22 +72,54 @@ def test_check_space():
         sys.stderr = save_stderr
 
 
-def test_check_tablespace():
-    outfile = utils.get_test_data('truncated.fq')
-    save_stderr, sys.stderr = sys.stderr, io.StringIO()
+@pytest.mark.parametrize('graph_type,buckets_per_byte', [
+    ('countgraph', 1),
+    ('smallcountgraph', 2),
+    ('nodegraph', 8),
+])
+def test_check_tablespace(graph_type, buckets_per_byte):
+    oldstderr = sys.stderr
+    sys.stderr = StringIO()
```

Comment:
There is a `capfd` fixture in pytest that takes care of capturing stdout/err.

```
capfd -- ..
    Enable capturing of writes to file descriptors 1 and 2 and make
    captured output available via ``capfd.readouterr()`` method calls
    which return a ``(out, err)`` tuple.
```

potentially less brittle than our homebaked solution. Not quite sure what the difference is to the `capsys` fixture.